todo-backend/TodoRedis/function.js

```diff
@@ -0,0 +1,40 @@
+const Redis = require('ioredis');
+
+const {
+  REDIS_PORT: port,
+  REDIS_HOST: host,
+  REDIS_PWD: password,
+} = process.env;
+
+const redis = new Redis({
+  port,
+  host,
+  family: 4,
+  password,
```

Comment:
change to `pwd`

Reply:
`Redis` expects the option called `password`. I changed it to use the env name directly: `  password: REDIS_PWD,`